lib/duo.c

```diff
@@ -767,7 +775,7 @@ duo_sync_time_offset(struct duo_ctx *ctx) {
     duo_code_t ret;
 
     ctx->argc = 0; /* no params */
-    ret = duo_call(ctx, "GET", DUO_AUTH_PING_ENDPOINT, ctx->https_timeout);
+    ret = duo_call(ctx, "GET", DUO_API_VERSION "/ping", ctx->https_timeout);
```

Comment:
Is this typical string concatenation syntax in C?  it doesn't look familiar.

Reply:
It is string literal concatenation. It happens during the preprocessing stage, not at runtime. The compiler joins adjacent string literals into one